resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -16,6 +16,14 @@ stages:
           - 'ci-it-backend'
       mavenExecuteIntegration:
         filePattern: 'integration-tests/pom.xml'
+  compliance:
```

Comment:
Wouldn't it be possible to keep the configuration compatible if we still use `sonarQubeScan` as the stage name in Cloud SDK Pipeline and add `sonarProperties` as an alias for the `options` configuration parameter of the `sonarExecuteScan` step?

Reply:
Not 100% sure, but I think the idea is that more steps/stages would be migrated to the Compliance stage, for example WhiteSource. The distinction "security versus compliance" of the tools is somewhat fuzzy, so I am not completely sure, but I think that is the reason for not keeping the stage name.